src/app/pages/AccountPage/Features/TransactionHistory/index.tsx

```diff
@@ -35,6 +39,27 @@ export function TransactionHistory(props: Props) {
 
   return (
     <Box gap="medium" margin="none">
+      <AlertBox
+        status="ok-weak"
+        justify="between"
+        endSlot={
+          <span>
+            <Anchor
+              href={''}
+              target="_blank"
+              rel="noopener"
+              color="brand"
+              style={{ verticalAlign: 'middle' }}
+            >
+              <Text size="xsmall">{t('account.summary.viewAccountTxs', 'View account transactions')}</Text>
+              <FormNext color="brand" size="18px" />
+            </Anchor>
+          </span>
+        }
+      >
+        {t('account.summary.someTxsInPendingState', 'Some transactions are currently in a pending state.')}
+      </AlertBox>
```

Comment:
Satisfied my use case in #1954 
![image](https://github.com/oasisprotocol/oasis-wallet-web/assets/9722540/244cdc95-10b9-4213-bb05-91832be41c1a)